etl/topline_summary.kp/knowledge.md

```diff
@@ -0,0 +1,447 @@
+---
+title: Topline Summary using Main Summary Dataset
+authors:
+- acmiyaguchi
+tags:
+- main_summary
+- executive
+- topline
+- summary
+- dataframe
+- spark
+- joins
+created_at: 2016-11-14 00:00:00
+updated_at: 2016-11-30 16:34:01.126578
+tldr: 'Port of the topline/executive summary using the main summary dataset (not for
+  production usage).
+
+  '
+---
+# Topline Summary using Main Summary Dataset
+
+[Bug 1309574](https://bugzilla.mozilla.org/show_bug.cgi?id=1309574) involves porting the executive summary to use the main summary dataset. The original script can be found [mozilla-services/data-pipeline/run_executive_report.py](https://github.com/mozilla-services/data-pipeline/blob/e5c29541794325388336a210746029dce998b9e5/reports/executive_summary/run_executive_report.py).
```

Comment:
Please also add a comment indicating, for each field in the expected output, whether that field is implemented here or not.